docs/design/sql/05-client-support.md

```diff
@@ -127,31 +126,23 @@ request {
 
 response {}
 ```
+
+The server-side resources are cleared automatically when the end of data is reached. Therefore, it is not necessary to send
+the `cancel` command to the member, if the previous `execute` or `fetch` command returned the last page, or an update count.
```

Comment:
```suggestion
the `close` command to the member, if the previous `execute` or `fetch` command returned the last page, or an update count.
```

Reply:
Fixed in https://github.com/hazelcast/hazelcast/pull/18182/commits/16835b52ccc39bc6d47dd05ffccc511d8a747be3